### PR TITLE
Coach team info

### PR DESCRIPTION
**Pull request by IdoDePro**
basic coach team info still need to add auto data maybe?

### CODE REVIEW

File: lib/views/mobile/screens/coach_team_info/coach_team_info_line_charts.dart [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import "package:scouting_frontend/models/team_data/team_data.dart";
+
+class CoachTeamInfoLineCharts extends StatelessWidget {
+  CoachTeamInfoLineCharts(this.data);
+  final TeamData data;
+  @override
+  Widget build(final BuildContext context) => CarouselWithIndicator(
+        direction: Axis.vertical,
+        enableInfininteScroll: true,
+        //TODO add linecharts
```
Comment (itamarsch) [inline lib/views/mobile/screens/coach_team_info/coach_team_info_line_charts.dart:12]: TODO?

File: lib/views/mobile/screens/coach_team_info/coach_team_info_data.dart [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        body: FutureBuilder<TeamData>(
+          future: fetchSingleTeamData(
+            team.id,
+            context,
+          ), //fetchTeam(team.id, context),
+          builder: (
+            final BuildContext context,
+            final AsyncSnapshot<TeamData> snapshot,
+          ) {
+            if (snapshot.hasError) {
```
Comment (itamarsch) [inline lib/views/mobile/screens/coach_team_info/coach_team_info_data.dart:34]: mapSnapshot?

File: lib/views/mobile/screens/coach_team_info/coach_team_info_data.dart [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                                    summaryData: data.summaryData,
+                                  ),
+                          ),
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.all(10),
+                          child: DashboardCard(
+                            title: "Quick data",
+                            body: CoachQuickData(
+                              QuickData(
```
Comment (itamarsch) [inline lib/views/mobile/screens/coach_team_info/coach_team_info_data.dart:77]: What happened to QuickData from TeamData?

File: lib/views/mobile/screens/coach_team_info/coach_quick_data.dart [hunk 4]
```diff
@@ -0,0 +1,169 @@
+import "package:flutter/material.dart";
+import "package:scouting_frontend/models/team_info_models/quick_data.dart";
+
+class CoachQuickData extends StatelessWidget {
+  const CoachQuickData(this.data);
+  final QuickData data;
+
+  @override
+  Widget build(final BuildContext context) => data.amoutOfMatches == 0
```
Comment (itamarsch) [inline lib/views/mobile/screens/coach_team_info/coach_quick_data.dart:9]: Try to find a way to combine the quick data in team info and we can probably remove some fields

File: lib/views/mobile/screens/coach_team_info/coach_auto_data.dart [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                  "slot 1", //TODO rename
+                  style: TextStyle(fontSize: 18),
+                ),
+              ),
+              data.slot1Data.amoutOfMatches == 0
+                  ? const Center(child: Text("No data"))
+                  : const Column(
+                      mainAxisSize: MainAxisSize.min,
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[], //TODO add season specific data
```
Comment (itamarsch) [inline lib/views/mobile/screens/coach_team_info/coach_auto_data.dart:35]: What is this file?

**Review by itamarsch — CHANGES_REQUESTED** [review]

File: lib/views/mobile/screens/coach_team_info/coach_auto_data.dart (hunk 5 above)
Comment (IdoDePro) [reply lib/views/mobile/screens/coach_team_info/coach_auto_data.dart:35]: not relevant yet until chayko finishes auto screen

**Review by itamarsch — APPROVED** [review]

**Pull request merged**